core/include/kernel/transfer_list.h

```diff
@@ -0,0 +1,126 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023, Linaro Limited
+ */
+
+#ifndef __KERNEL_TRANSFER_LIST_H
+#define __KERNEL_TRANSFER_LIST_H
+
+#define TRANSFER_LIST_SIGNATURE		U(0x006ed0ff)
```

Comment:
From spec v0.9, it seems the expected signature value is 0x0040b10b.

Reply:
Did you get the spec from this link? https://github.com/FirmwareHandoff/firmware_handoff/releases/tag/v0.9
Please see the Table1, "signature | 0x4 | 0x0 | The value of signature must be 0x6e_d0ff“.